airbyte/_util/api_util.py

```
@@ -720,16 +740,253 @@
+        client_id=client_id,
+        client_secret=client_secret,
+        api_root=config_api_root,
+    )
+    headers: dict[str, Any] = {
+        "Content-Type": "application/json",
+        "Authorization": SecretString(f"Bearer {bearer_token}"),
+    }
+    request_body: dict[str, str] = {
+        # https://github.com/airbytehq/airbyte-platform-internal/blob/10bb92e1745a282e785eedfcbed1ba72654c4e4e/oss/airbyte-api/server-api/src/main/openapi/config.yaml#L7376
+        "value": override.docker_image_override,
+        "config_key": "docker_image",  # TODO: Fix this.
```

Ruff (0.8.2)

Missing issue link on the line following this TODO

(TD003)